content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -30,6 +30,21 @@ Specifically, Odoo's predictive lead scoring uses the *naive Bayes* probability
    P(A | B) = \frac{P(A) \times P(B | A)}{P(B)}
    \end{equation}
 
+Breaking down the equation:
+
+- P(A|B) = The probability of a successful lead *in this case*
+- P(A) = The overall probability of a lead being successful regardless of the conditions
+- P(B|A) = The probability of this being the case given a lead is successful
+- P(B) = The probability of this being the case
+
+The term *in this case* is referring to the variables that can affect a lead being
+successful in Odoo. This can include variables such as the assigned Salesperson, the source of the
+lead, the language of the lead, and other historical and demographic data.
+
+Which variables are considered in this calculation can be :ref:`configured
+<lead_scoring/configuration>` to tailor the calculation to
+each business's needs.
```

Comment:
Double check your <100 char line breaks using the `make review` command in your terminal (many of these broke too early 😉 )